.github/workflows/ci.yml

```diff
@@ -0,0 +1,40 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: Node.js v${{ matrix.nodejs }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        nodejs: [12, 14]
+    env:
+      CI: true
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.nodejs }}
+
+    - name: Install
+      run: |
+        npm ci
```

Comment:
Shouldn't this be `npm run test`? I don't see a "ci" script in the `package.json`. 🤔

Reply:
it's like `npm install` except making sure to install from package-lock.json and not package.json. This makes the installation faster and ensures package-lock.json is in-sync with package.json :)

https://docs.npmjs.com/cli/v7/commands/npm-ci

Reply:
Ah, nice. 💡 Didn't know this one yet. 💚